website/content/en/preview/getting-started/cloudformation.yaml

```diff
@@ -48,6 +48,7 @@ Resources:
               - ec2:CreateTags
               - iam:PassRole
               - ec2:TerminateInstances
+              - ec2:DeleteLaunchTemplate
```

Comment:
We should make some upgrade instructions to help users migrate.

Reply:
we definitely should... not sure how we'd structure them. I was thinking release notes, but another way would be adding an upgrade section in the versioned docs.

Reply:
I just want a single command to run which updates my IAM. Maybe cfn deploy works out of the box.

Reply:
yeah cfn should work fine, but I suspect most users are integrating our template (cfn or terraform or cdk) into their own infrastructure-as-code so there won't be a one-size fits all solution.